### PR TITLE
okular: refresh page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** latest

### CODE REVIEW

**Review by blueskyson — APPROVED** [review]

File: pages/common/okular.md [hunk 1]
```diff
@@ -1,25 +1,25 @@
...
+- Open specific documents:
 
 `okular {{path/to/file1 path/to/file2 ...}}`
 
 - Open a document at a specific page:
 
 `okular --page {{page_number}} {{path/to/file}}`
 
-- Open a document in presentation mode:
+- Open a specific document in a presentation mode:
```
Comment (kbdharun) [inline pages/common/okular.md:18]:
```suggestion
- Open a specific document in presentation mode:
```
I don't `a` will come here.

File: pages/common/okular.md [hunk 2]
```diff
@@ -1,25 +1,25 @@
 # okular
 
-> A universal document viewer.
+> View documents.
 > More information: <https://okular.kde.org/>.
 
-- Launch Okular:
+- Launch a document viewer:
```
Comment (kbdharun) [inline pages/common/okular.md:6]:
```suggestion
- Launch document viewer:
```
Same here

File: pages/common/okular.md [hunk 3]
```diff
@@ -1,25 +1,25 @@
 # okular
 
-> A universal document viewer.
+> View documents.
 > More information: <https://okular.kde.org/>.
```
Comment (kbdharun) [inline pages/common/okular.md:4]:
```suggestion
> More information: <https://docs.kde.org/stable5/en/okular/okular/command-line-options.html>.
```
Optional suggestion.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**